pyproject.toml

```diff
@@ -32,6 +32,10 @@ asynctest = "~0.13"
 pytest = "~6.2"
 pytest-cov = "~2.8"
 pytest-lazy-fixture = "~0.6"
+Sphinx = "^4.4.0"
```

Comment:
only if we want to distribute documentation with the guix package. it would be nice since we could also automatically generate `.deb` packages. otherwise we patch out these dependencies from `pyproject.toml` inside the guix package.